datareduction/datareductionoperator.cpp

```diff
@@ -1554,7 +1492,7 @@ namespace DRO {
    // Split into VariablePTensorBackstreamDiagonal (11, 22, 33)
    // and VariablePTensorOffDiagonal (23, 13, 12)
 
-   VariableMinValue::VariableMinValue(): DataReductionOperator() { }
+   VariableMinValue::VariableMinValue(): DataReductionOperatorHandlesWriting() { }
    VariableMinValue::~VariableMinValue() { }
```

Comment:
I don't remember why we have two different mechanisms, min/max f value for diagnostic but only min with a different DRO for output. Can't we merge them? And extend to have max as well for output?

Reply:
Hmm, the spatially resolved minValue datareducer runs spatialCell::getVelocityBlockMinValue(), whereas the MinDistributionFunction datareducer iterates through the block data itself. While getVelocityBlockMinValue() should be much faster (since it is simply a pre-stored value), it might be more reasonable to use the explicit iteration process just in case we want to catch bugs in exactly that calculation. Not sure which one I like more.

Reply:
We could join min and max and have that in one pass?